test-app/src/main.ts

```diff
@@ -43,7 +39,9 @@ if (btr) {
 } else {
 	const nodeBtrCache = document.createElement('div');
 	nodeBtrCache.id = 'nodeBtrCache';
-	nodeBtrCache.innerHTML = test('./src/foo.txt');
+	test('./src/foo.txt').then((result: string) => {
```

Comment:
@matt-gadd All blocks are now async so had to set the result from the resolved promise